README.md

```diff
@@ -18,6 +18,23 @@ npm install ember-statechart-component
 yarn add ember-statechart-component
 ```
 
+### Optional
+
+To be able to use XState [`state.matches`](https://xstate.js.org/api/classes/state.html#matches)
```

Comment:
I got a little bit tangled in English sentence structure with the "which is available on the `state` object", so instead I just merged in the information here. It is quite a good term to grep for later on in the README, so it seems ok to me. Happy to change.